packages/editor-ui/src/components/OnboardingCallSignupModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+	<Modal
+		:name="ONBOARDING_CALL_SIGNUP_MODAL_KEY"
+		:title="$locale.baseText('onboardingCallSignupModal.title')"
+		:eventBus="modalBus"
+		:center="true"
+		:showClose="false"
+		:beforeClose="onModalClose"
+		width="460px"
+	>
+		<template slot="content">
+			<div class="pb-m">
+				<n8n-text>
+					{{ $locale.baseText('onboardingCallSignupModal.description') }}
+				</n8n-text>
+			</div>
+			<div @keyup.enter="onSignup">
+				<n8n-input v-model="email" :placeholder="$locale.baseText('onboardingCallSignupModal.emailInput.placeholder')" />
+				<div v-if="showError" :class="[$style.error, 'mt-4xs']">
+					<n8n-text size="small">
+						{{ $locale.baseText('onboardingCallSignupModal.infoText.emailError') }}
+					</n8n-text>
+				</div>
```

Comment:
nitpick - I think this would be same thing.. you don't have to change this
```suggestion
					<n8n-text v-if="showError" size="small" class="mt-4xs" tag="div" color="danger">
						{{ $locale.baseText('onboardingCallSignupModal.infoText.emailError') }}
					</n8n-text>
```

Reply:
Good catch. `error` class used to have some other props which I deleted in the final version so it's not needed anymore.